### DISCUSSION TITLE
temux

### DISCUSSION THREAD

**Discussion by MatejMagat305** [fyne-io/fyne#3850] [Q&A]
well I found way how to make apk in termux (and finally it is working), should I reopen add termux available #3649 or open new PR? And can I create video and script similar to  https://youtu.be/7aUh39w_-2Q and https://github.com/MatejMagat305/vab-termux? Of course I will put "fyne" to title of video ...

**Comment by andydotxyz** [comment]
Thanks. Usually it is best to open a new PR.

**Reply by MatejMagat305** [reply]
here is link: [video](https://youtu.be/uGtVjf4_Ivo) and [script](https://github.com/MatejMagat305/fyne-termux) script contains link to my fork, but I hope I will change soon ...

**Reply by Bluebugs** [reply]
You closed your PR, but I couldn't understand why. It did look of to me.

**Reply by MatejMagat305** [reply]
so it have not working until now (it taked some time ) and I open  #3851 as was advise here ...

**Reply by ddkwork** [reply]
I was maked a test.

**Reply by ddkwork** [reply]
【termux build apk ok，but function error-哔哩哔哩】 https://b23.tv/KHp91A3

**Reply by andydotxyz** [reply]
Your issue is unrelated - there is no content to the window as far as I can see... you call:

`w.SetContent(fyne.CanvasObject(w))` which is going to fail - I'm surprised the app doesn't panic.

**Comment by andydotxyz** [comment]
Thanks @MatejMagat305 this is added on `develop` and we should be able to include it on 2.4.0 in June.

**Reply by MatejMagat305** [reply]
thank for accept ..., after release 2.4.0 I will update my [script](https://github.com/MatejMagat305/fyne-termux) to install fyne tool from your repo ... by clasical `go install ...`
